src/app/components/home/home.component.html

```diff
@@ -9,116 +9,58 @@
 <mat-drawer-container class="sidenav-container">
   <mat-drawer #drawer mode="side" align="start" disableClose="false" opened>
     <mat-nav-list [disableRipple]="true">
-    <mat-accordion displayMode="flat">
-      <mat-expansion-panel (opened)="setOfficeStep(1)" hideToggle="true">
-        <mat-expansion-panel-header [ngClass]="{ 'active-link' : officeStep === 1 }" collapsedHeight="50px" expandedHeight="50px" routerLink="/mayor">
-          <mat-panel-title>
-              Mayor
-          </mat-panel-title>
-        </mat-expansion-panel-header>
-        <ng-container *ngFor="let candidateType of candidates | keyvalue">
-          <ul *ngFor="let candidate of candidateType.value.candidates | keyvalue; let i = index">
-            <li *ngIf="candidateType.key === 'mayor'">
-              <a (click)="selectSidenavCandidate(candidate.key)">{{candidate.key}}</a>
-            </li>
-          </ul>
-        </ng-container>
-      </mat-expansion-panel>
+        <mat-accordion displayMode="flat">
 
-      <mat-expansion-panel (opened)="setOfficeStep(2)" hideToggle="true">
-        <mat-expansion-panel-header [ngClass]="{ 'active-link' : officeStep === 2 }" collapsedHeight="50px" expandedHeight="50px" routerLink="/city-attorney">
-          <mat-panel-title>
-              City Attorney
-          </mat-panel-title>
-        </mat-expansion-panel-header>
-        <ng-container *ngFor="let candidateType of candidates | keyvalue">
-          <ul *ngFor="let candidate of candidateType.value.candidates | keyvalue; let i = index">
-            <li *ngIf="candidateType.key === 'city-attorney'">
-              <a (click)="selectSidenavCandidate(candidate.key)">{{candidate.key}}</a>
-            </li>
-          </ul>
-        </ng-container>
-      </mat-expansion-panel>
+            <ng-container *ngFor="let candidateType of sortedObj | keyvalue: asIsOrder; let i= index">
+             
+              <!-- <ng-template #cityCouncilBlock> -->
+              <mat-expansion-panel (opened)="setOfficeStep(i)" hideToggle="true" (closed)="candidateType.key.toLowerCase() ==='city council'?setCouncilDistrictStep(0): otherOfficeClicked(candidateType.key)">
```

Comment:
It would be nice to someday figure out how to avoid any kind of conditionals on the actual data values. Just City Council is kind of annoying because there are several

Reply:
will think over it tonight, to further improve and possibly make the dom agnostic.